cli/src/main/java/org/jboss/galleon/cli/HelpSupport.java

```diff
@@ -168,6 +89,134 @@ private static String pad(int max, String k) {
         return lst;
     }
 
+    public static String buildHelp(CommandRegistry<? extends Command, ? extends CommandInvocation> registry,
+                                    Set<String> commands) throws CommandNotFoundException {
+        return buildHelp(registry, commands, false);
```

Comment:
shouldn't it be true? Called from the HelpCommand we do want the footer, no?

Reply:
Blimey, yes. :(

Reply:
I can't comment on the other one, but I just investigated and no, we're not safe.

For example:
```
[cli]$ help maven set-local-repository
Usage: set-local-repository Set the path to the local maven repository path

Argument:
   Path to local maven repository
```
and
```
== Commands to configure CLI tool ==
maven : Contains commands to configure maven support
    add-repository : Add a maven repository
        --enable-release Enable or disable 'release' artifacts resolution
        --enable-snapshot Enable or disable 'snapshot' artifacts resolution
        --name Maven remote repository name
        --release-update-policy Maven release update policy
        --snapshot-update-policy Maven snapshot update policy
        --type Maven remote repository type, "default" by default
        --url Maven remote repository URL
    enable-offline : Enable or disable the offline mode
    enable-release : Enable or disable 'release' artifacts resolution
    enable-snapshot : Enable or disable 'snapshot' artifacts resolution
    get-info : Display maven configuration content
    remove-repository : Remove a maven repository from configuration
    reset-local-repository : Reset to the local repository default path
    reset-offline : Reset the offline mode to its default value
    reset-release : Reset release artifact resolution to default value
    reset-release-update-policy : Reset to the default release update policy
    reset-settings-file : Reset the path to the maven xml settings file
    reset-snapshot : Reset snapshot artifact resolution to default value
    reset-snapshot-update-policy : Reset to the default snapshot update policy
    resolve-feature-pack : Download a feature pack to the local maven repository
        <arg> : Location of feature pack to resolve
        --verbose Whether or not the output should be verbose
    set-local-repository : Set the path to the local maven repository path
    set-release-update-policy : Set the default release update policy
    set-settings-file : Set the path to the maven xml settings file
    set-snapshot-update-policy : Set the default snapshot update policy
```